### ISSUE TITLE
v0.0.2 and Notebook running in JupyterLite

### ISSUE THREAD

**Issue by ns-rse** [Mesnage-Org/pgfinder#151]
Once #145 is complete the next step is to make a release (`v0.0.2`) which will remove the dependency on `pysqlite3` that currently exists in `v0.0.1`. This dependency prevents installing `pgfinder` in a [JupyterLite](https://github.com/jupyterlite/jupyterlite) notebook which is currently the preferred solution for solving issues that have arisen with Binder (see [here](https://blog.jupyter.org/mybinder-org-reducing-capacity-c93ccfc6413f)).

## Resources for JupyterLite...

* [JupyterLite](https://github.com/jupyterlite/jupyterlite)
* [JupyterCon 2023 - JupyterLite](https://jtp.io/jupytercon-2023-jupyterlite/files/index.html)

## Installing (or not) from GitHub

There is no filesystem available in Pyodide and so its not possible to install the development version of `pgfinder` from GitHub for testing of new features.

Certain requirements are also required of released packages to PyPI see [FAQ - Why can't Micopip find a "pure Python wheel" for a package?](https://pyodide.org/en/stable/usage/faq.html#micropip-can-t-find-a-pure-python-wheel).

This may present challenges for development but it will always be possible to test Notebooks from locally (i.e. on own computer) launched system and we will still be able to have the Binder Notebooks that run from development available (its just that Binder now regularly lacks resources to start instances, see above linked issue).

**Comment by bobturneruk** [comment]
Drat. Can you just drop the dependency and try again, see if this is pervasive?

**Comment by ns-rse** [comment]
Not without removing the `find_pg` entry point I added to give a CLI.

I've made the switch to `pyYAML` which was considerably less painful than I expected and all tests pass. I just need to check if I can release something like `v0.0.3-alpha1` and have it uploaded to PyPI as I'd rather avoid releasing `v0.0.3` only to find there is another dependency that doesn't provide a `.whl` and won't install under Pyodide that needs working around.
